### ISSUE TITLE
Feature request: ability to open multiple files at once

### ISSUE THREAD

**Issue by hubalazs** [thonny/thonny#1154]
Currently the file open dialog allows only one file to be selected and opened. This needs to be extended to multiple files.

**Comment by aivarannamaa** [comment]
Good idea! In the meanwhile, many files can be easily opened via Thonny's file browser (View => Files)

**Comment by EkremDincel** [comment]
Files tab has "Create Directory" option, It would be good if it has "Create File" option too.

**Comment by aivarannamaa** [comment]
> Files tab has "Create Directory" option, It would be great if it has "Create File" option too.
